automatic/opera/tools/chocolateyInstall.ps1

```diff
@@ -18,7 +15,7 @@ $packageArgs = @{
   validExitCodes = @(0)
 }
 
-if (IsVersionAlreadyInstalled $version) {
+if (!$Env:ChocolateyForce -and (IsVersionAlreadyInstalled $Env:ChocolateyPackageVersion)) {
```

Comment:
using `$env:ChocolateyPackageVersion` won't work in the long run, the package version may be different than the software version.

Reply:
In cases where we need to provide a package fix.

Reply:
OK, returned

Reply:
thanks